crates/bevy_render/src/render_resource/uniform_buffer.rs

```diff
@@ -156,7 +165,8 @@ impl<T: ShaderType> Default for DynamicUniformBuffer<T> {
             buffer: None,
             capacity: 0,
             label: None,
-            label_changed: false,
+            changed: false,
+            buffer_usage: BufferUsages::COPY_DST | BufferUsages::UNIFORM,
```

Comment:
These defaults need to be documented, and the latter use of`BufferUsages::UNIFORM` should be strictly enforced.

Reply:
Should I put the BufferUsages::UNIFORM in write_buffer method?